### DISCUSSION TITLE
Contributing new ideas for DAOs

### DISCUSSION THREAD

**Discussion by thelastjosh** [metagov/daostar#40] [Ideas]
DAOs are not only smart contracts; they are an open-source approach to organizational design. 

So if you have idea for DAOs that you think the **entire** ecosystem could benefit from, but... 

- you want some feedback before you write up a whole new DAOIP
- you're not sure if it's been done before
- you're looking for collaborators (technical, legal, academic, etc.—organizations are complicated systems!)

Then you've come to the right place! Post your idea / feature request under the "Idea" category at https://github.com/metagov/daostar/discussions/categories/ideas.

### DAO Creative Universe Instructions

1. If you have an idea or suggestion, post it as a discussion thread in [the Ideas category](https://github.com/metagov/daostar/discussions/categories/ideas).
2. Note the number of the thread, e.g. 40 for `https://github.com/metagov/daostar/discussions/40`. This will be your idea number.
3. Print a sticker on the provided sticker paper.
4. Grab a large (2 inch) medallion, attach your sticker, and write down the idea number on the back of the medallion. 
5. Add your medallion to the DAO Creative Universe!

DO NOT post ideas here if you have no plans to write them up or implement them as a DAOIP. If you have a feature request for the DAOstar API infrastructure that you want the DAOstar editors / engineering team to implement, post that [as an issue](https://github.com/metagov/daostar/issues) in this repo.

**Comment by JJN2808** [comment]
Hi Joshua, 

I'm not yet sure how Github works, since I just joined, but I hope it is okay to come with a request here. 

I'm writing a bachelor thesis about DAOs with two co-students. We all study at Copenhagen Business School where we are enrolled in a program called BSc in Business Administration and Sociology. We are in the very beginning of the thesis and would really like to get in touch with some people who know stuff about DAOs and can help us grasp their complexity. Would you by any chance be willing to answer a few questions or do you know someone who might would? We are still not completely settled on the direction of the thesis, but we think that we want to look into the organisational aspect, what drives people to join a DAO instead of a traditional organisation and the community within and around DAOs. Thank you in advance!

Best regards, 
Julie Nielsen

**Reply by thelastjosh** [reply]
Hi Julie, I think a good place to start is to simply join some DAOs and ask! If you're doing research on DAOs, I'd encourage to take a look at [Metagov](metagov.org) and consider joining the Slack.

**Comment by Tuxman** [comment]
Would really love a way to figure out how DAO's can own real assets e.g. real estate. ATXDAO is working on the logistics and framework for how we might have a permanent hacker house in Austin owned by the DAO directly.
